### ISSUE TITLE
Øret → Ører

### ISSUE THREAD

**Issue by danmichaelo** [realfagstermer/realfagstermer#1211]
Siden vi har to.

Jfr. Øyne, https://app.uio.no/ub/emnesok/humord/search?id=c03898

**Comment by kyrretl** [comment]
Hm, jeg føler egentlig at det bør være Øret, siden det er snakk om et biologisk organ, men jeg ser at vi har Nyrer i flertall (noe som høres noget rart ut, vanlige mennesker sier vel Nyrene), men Nyren høres jo ikke bra ut.
 
https://app.uio.no/ub/emnesok/realfagstermer/search?id=c007486

Apropos, øyne/øyet er vi ikke konsekvente mph nynorsk bokmål:
https://app.uio.no/ub/emnesok/realfagstermer/search?id=c006305

LC har det som entall:
http://id.loc.gov/search/?q=eye&q=cs%3Ahttp%3A%2F%2Fid.loc.gov%2Fauthorities%2Fsubjects
http://id.loc.gov/search/?q=ear&q=cs%3Ahttp%3A%2F%2Fid.loc.gov%2Fauthorities%2Fsubjects

Mesh har også øye entall: http://mesh.uia.no/ (må søke selv), mens øre er i ubestemt form entall...

Det ser ut som vi egentlig har en oppryddingsjobb mhp. flertall/entall og bør kanskje få en klarhet i forhold til tvilstilfellene. Når det gjelder selve emnesøket er det ikke så farlig siden man får treff i BF uansett, men vi bør bli enige om hva som skal være hovedterm.

Hva slags retningslinjer har vi? Kan vi lage våre egne?

**Comment by Hypsibius** [comment]
Jo, men det er kanskje forvirrande at auga /auget kan ha to kjønn på bokmål: 
![image](https://user-images.githubusercontent.com/9350370/38865384-e0992a7e-423d-11e8-8e50-0d7eb36c3c73.png)
